website-1.x/versioned_docs/version-1.8.0/api-commands.md

```diff
@@ -147,6 +147,7 @@ This command will build the static website, apply translations if necessary, and
 | Options           | Default | Description                                                                                                                          |
 | ----------------- | ------- | ------------------------------------------------------------------------------------------------------------------------------------ |
 | `--port <number>` | `3000`  | The website will be served from port 3000 by default, but if the port is taken up, Docusaurus will attempt to find an available one. |
+|`--host <number>`|`localhost`|Use a custom host with localhost as the default.|
```

Comment:
why is it host <number> ?

Reply:
Per https://github.com/facebook/Docusaurus/pull/1475/files/326d299e5baeaee3f7dc6791eb96bf05de82767a#discussion_r284958920

Reply:
Oops my bad. I mean why `host <number>`.

GitHub swallowed the `<number>`

![image](https://user-images.githubusercontent.com/17883920/58035562-a27f4180-7b5b-11e9-9b2e-e8d534ef4de2.png)

Reply:
Ha. All good. Ya, I figured number as in 127.0.0.1 but maybe 'value' is better?

Reply:
Follow v2

https://github.com/facebook/Docusaurus/blob/382b88bb49cc13adb920dbb0b901afa44921b041/packages/docusaurus/bin/docusaurus.js#L82